bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -295,7 +295,7 @@ end
 rules.when().item('Apartment_MotionSensor').changed().to('ON').then(() => {
   actions.notificationBuilder('Motion detected in the apartment!')
     .withIcon('motion')
-    .withTag('motion-tag')
+    .withTag('Motion Tag')
     .withTitle('Motion Detected')
     .withReferenceId('motion-id-1234')
     .withMediaAttachment('https://apartment.my/camera-snapshot.jpg')
```

Comment:
@jimtng Can you please do me the favor and change the next line to:

```js
    .addActionButton('Turn on the light', 'command:Apartment_Light:ON')
```

Reply:
done

Reply:
Thanks!